jar.properties

```diff
@@ -30,8 +30,8 @@ slf4j-api.jar=lib/slf4j-api-1.7.6.jar
 slf4j-api.src=lib/slf4j-api-1.7.6-sources.jar
 slf4j-log4j.jar=lib/slf4j-log4j12-1.7.6.jar
 slf4j-log4j.src=lib/slf4j-log4j12-1.7.6-sources.jar
-logback-classic.jar=lib/logback-classic-1.1.1.jar
-logback-core.jar=lib/logback-core-1.1.1.jar
+logback-classic.jar=lib/logback-classic-1.1.2.jar
```

Comment:
jar.properties is actually not used anymore. Versionning of dependencies is done only in pom.xml. I should cleanup that jar.properties file.